gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/CertificateService.groovy

```diff
@@ -44,13 +44,13 @@ class CertificateService {
 
   List<Map> getCertificates(String selectorKey) {
     listCommand("certificates") {
-      clouddriverServiceSelector.select(selectorKey).getCertificates()
+      clouddriverServiceSelector.select().getCertificates()
```

Comment:
so in places where we used to pass the `selectorKey` will the selection behavior change? do we care? (I assume the selector was always the app which is not in the authcontext)

Reply:
the selector was the "user origin" app, which is in the auth context. You can get the same behavior as before with a config like this:

```
config:
  selectorClass: com.netflix.spinnaker.kork.web.selector.ByUserOriginSelector
  origin: deck
```

Reply:
and the `clouddriverServiceSelector` factory method does the translation, so you get opted into the new selector if you had the previous style of configuration:

```
dynamicEndpoints:
  deck: url
```